docs/docs/guides/tutorials/Nextjs.mdx

```diff
@@ -0,0 +1,750 @@
+# Using Hanko for a Todo app with Next.js 13 and Prisma
+
+In this tutorial, you’ll learn how to build a Todo app with the Next.js 13 popular “App Router” structure and understand some of the most important changes that come with it. We will build a fully functional todo app, handling the creation, updating when completed and the option to delete the todos.
+
+## Let’s set it up 
+At Hanko, we understand that finding the right stack for the project is a big step. In this guide, we will bring Next.js as the main character of the project, we will test  the Client vs Server Components. For the style, we will use Tailwind CSS. We will use Hanko for the login and registration, user management and logout. Prisma will handle the storage.
+
+### Initiliaze the next app
+To create a new Next.js app, we can use the `create-next-app` or `create-next-app@latest` command-line tool followed by the name of your choice for the app. Open your terminal in Visual Studio Code and run the following command: 
+
+```
+ npx  create-next-app@latest todo-nextjs-hanko 
+```
+
+Then you’ll  be asked some prompts on what you will use for the app. The project configuration options should look something like this:
+
+![Pre-config Next.js prompts](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/8djxl7jl42xl90wpps1b.png)
+
+The above choices will create a new Next.js app with the chosen name, all the required dependencies for this project will also be installed.
+
+#### Understanding the project structure
+When using the version 13 of  Next.js, we have the option to work with the App Router directory instead of the Pages Router, for a quick summary  we could say that:
+
+* The new directory named “app” is replacing “pages”
+* “page.tsx|page.jsx” is the new “index.tsx|index.jsx” 
+* “layout.tsx” is the new “_app.tsx”
+* Everything is a Server Component unless you make it a Client Component using the “use client” directive at the top of the file.
+* API Routes are now Server Components or Route Handlers (... More info on how this is  very important)
+
+Remove unnecessary files, such as logos, icons, etc. If you are going to use Tailwind CSS make sure to bring your desired configuration to the `tailwind.config.ts` file, defining your color palette, fonts, breakpoints, ect.
+
+ℹ️ For more information about the App Router of Next.js click [here](https://nextjs.org/docs).
+
+### Get Prisma started
```

Comment:
Instead of 'Get Prisma started', should we have something like, "Setting up prisma"? What do you think?